### DISCUSSION TITLE
Welcome to Dot-Batch_WebDevelopment-Notes Discussions!

### DISCUSSION THREAD

**Discussion by priyansh70** [priyansh70/Dot-Batch_WebDevelopment-Notes#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mohitkhichar** [comment]
Bhai studyNotion wale project ka documentation(pdf file) send kardo muje taaki project samajne m easy ho jaye muje
Email : mkhichar585@gmail.com

**Comment by AjinkyaBTambade** [comment]
I don't have anything bro sorry
